src/components/Main/Main.jsx

```diff
@@ -0,0 +1,28 @@
+import {useState, useEffect, useContext} from 'react'
+import './Main.css'
+import steamSVG from "../../../public/assets/images/steam.svg";
+import {AnimatePresence, motion} from "framer-motion";
+function Main({navigate}) {
+
+  const handleButtonClick = () => navigate('/libraries');
+
+  return (
+    
+      <div className="main">
+            <h2 className='main__subtitle'>NEW GAME?</h2>
+        <h2 className="main__continue" onClick={handleButtonClick}>click to continue</h2>
+        <AnimatePresence>
+          <motion.img
+            initial={{opacity: 0, y: 200}}
```

Comment:
* You need to format the markup (and code) to show **indentation** and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`